anesthetic/samples.py

```diff
@@ -97,6 +97,14 @@ def __init__(self, *args, **kwargs):
                 self['weight'] = self.weight
                 self.tex['weight'] = r'MCMC weight'
 
+            self.set_automatic_limits()
+
+    def set_automatic_limits(self):
+        """Set all unassigned limits to min and max of sample."""
```

Comment:
Happy with name and docstring? Open for suggestions of improvement.

Reply:
Perhaps make it _set_automatic_limits (i.e. with an extra leading underscore), as it's not something that a user is expected to call by themselves, but could do if they wanted to.